samples/auth-user-sample/test/auth.test.js

```diff
@@ -0,0 +1,99 @@
+// /**
+//  * Copyright 2019 Google LLC
+//  *
+//  * Licensed under the Apache License, Version 2.0 (the "License");
+//  * you may not use this file except in compliance with the License.
+//  * You may obtain a copy of the License at
+//  *
+//  *   https://www.apache.org/licenses/LICENSE-2.0
+//  *
+//  * Unless required by applicable law or agreed to in writing, software
+//  * distributed under the License is distributed on an "AS IS" BASIS,
+//  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  * See the License for the specific language governing permissions and
+//  * limitations under the License.
+//  */
+
+// 'use strict';
+
+const mocha = require('mocha');
```

Comment:
Generally don't have to explicitly require mocha like this, or assign these below.  They're just ambiently available

Reply:
My linter says no-undefined when I don't do these assignments. Is there something else I should be doing about that?

Reply:
Yes! You can modify `samples/auth-user-sample/test/.eslintrc.yml to include:

```
---
env:
  mocha: true
```

This signals to `eslint` that you're using the ambient functions available in mocha, and will leave you alone :)

Reply:
Thanks!